### DISCUSSION TITLE
Allow URL param in app

### DISCUSSION THREAD

**Discussion by loan-talvat** [directus/directus#10555] [Feature Requests]
It would be great in Vue app to add parameters in GET to filter or search like in API

For example, two data models
- Car model
- Car brand

On a brand's page, create a button to open all models of that brand: /admin/collections{or}content/model?Filter[brand][_ eq]=GMC

In this way I directly access the listing of the models on the application with a filter already informed

**Comment by rijkvanzanten** [comment]
Would be great to sync the configured filters to the URL! We will have to take the maximum URL length into account. Once you start doing local operations with deeply nested filters, those query params can get quite long

**Reply by loan-talvat** [reply]
or not necessarily similar to the API, at least have the possibility of doing a simple search or a simple filter

**Reply by karamokoisrael** [reply]
You're right, I was wondering myself if it wasn't already existing

**Reply by loan-talvat** [reply]
to come back to what we were saying, the objective would be to make searches via URL possible, but not to pass everything via URL!
By default we keep the current behavior, but if we see that there are more parameters in the url, we add them

**Comment by LevinGermann** [comment]
Are there any plans to work on this?
It would be great, with this we can use Directus with the collections and link to it from external systems.
e.g. https://link/admin/content/review_photo?filter[tags][_contains]=summer

without having our users go to Directus, select the collection and type "summer" in the searchbar which does a full-collection scan for this word and is extra manual work.

**Comment by loan-talvat** [comment]
I think this can be very useful
Sync is not necessary, but, if we have a query, we can get it and filter..

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
